.github/workflows/release.yaml

```diff
@@ -14,14 +14,14 @@ jobs:
       - name: Check out
         uses: actions/checkout@v3
         with:
-          token: ${{ secrets.GITHUB_TOKEN }}
+          token: ${{ secrets.BOT_ACCESS_TOKEN }}
 
       - name: Check head commit message
         id: check_head_commit_message
         continue-on-error: true
         run: |
           head_commit_message=`git log --format=%s -1`
-          python ./.github/scripts/validate_release_commit.py $head_commit_message
+          python ./.github/scripts/validate_release_commit.py "$head_commit_message"
```

Comment:
why do we need the quotes?

Reply:
Yes, I think this is one of the main reason why it fails last time. It seems to be passed as `python ./.github/scripts/validate_release_commit.py Release 1.17.2 #123123` which causes an error